### DISCUSSION TITLE
Music Provider: Soundtrack Your Brand

### DISCUSSION THREAD

**Discussion by joealden** [music-assistant/support#2550] [Feature requests and ideas]
https://www.soundtrackyourbrand.com

**Comment by domasist** [comment]
would help businesses which are using Music Assistant / Home Assistant

**Comment by slrjacobk** [comment]
Would love to see this!

**Comment by Test-subj** [comment]
It's been a long time since this request happened.. they have a nice API and stuff now. 
https://api.soundtrackyourbrand.com/v2/docs

**Reply by OzGav** [reply]
Unfortunately not the API appears to only do control not content delivery

**Reply by Test-subj** [reply]
Yeah.. I started digging in to see if I could get something working after I posted that and quickly hit a wall. I'm in the middle of submitting a support request to them.

**Reply by Test-subj** [reply]
It seems possible to output audio from Soundtrack to MA using the alpha Airplay plugin as a bridge. Then the API would actually become useful to allow control from MA.

**Reply by Test-subj** [reply]
Soundtrack got back to me and said that any MA implementation would have to be done by them and that they have no interest in it at this time. At this point, I might just have the bar switch to Soundsuit since I'd be able to embed their web player in an iframe and call it good.
